### PR TITLE
Added Python Getting started tutorial

### PR DESCRIPTION
**Pull request by ahcorde**
Signed-off-by: ahcorde <ahcorde@gmail.com>
# 🎉 New feature

* Part of #210 

## Summary

Added Python Getting started tutorial

## Checklist
- [x] Signed all commits for DCO
- [ ] Added tests
- [x] Added example and/or tutorial
- [x] Updated documentation (as needed)
- [ ] Updated migration guide (as needed)
- [x] `codecheck` passed (See [contributing](https://ignitionrobotics.org/docs/all/contributing#contributing-code))
- [x] All tests passed (See [test coverage](https://ignitionrobotics.org/docs/all/contributing#test-coverage))
- [x] While waiting for a review on your PR, please help review [another open pull request](https://github.com/pulls?q=is%3Aopen+is%3Apr+user%3Aignitionrobotics+repo%3Aosrf%2Fsdformat+archived%3Afalse+) to support the maintainers

**Note to maintainers**: Remember to use **Squash-Merge** and edit the commit message to match the pull request summary while retaining `Signed-off-by` messages.

### CODE REVIEW

File: tutorials/pythongetstarted.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+import ignition.math
+
+def main():
+  pass
+
+if __name__ == "__main__":
+  main()
+```
+
+Now let's create to 3D points with arbitrary values. We will use the
```
Comment (chapulina) [inline tutorials/pythongetstarted.md:42]:
```suggestion
Now let's create two 3D points with arbitrary values. We will use the
```

File: tutorials/pythongetstarted.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+ignition.math.Vector3 class to represent these points. Ignition Math provides a handy
+ignition.math.Vector3d type which is a typedef of `Vector3d`. The result of this
```
Comment (chapulina) [inline tutorials/pythongetstarted.md:44]: Does the `typedef` sentence make sense for Python users? Is `ignition.math.Vector3` something that they can use? I think this paragraph can be reworded for Python.

File: tutorials/pythongetstarted.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+def main():
+  point1 = Vector3d(1, 3, 5)
+  point2 = Vector3d(2, 4, 6)
+
+if __name__ == "__main__":
+  main()
+```
+
+Finally, we can compute the distance between `point1` and `point2` using the
+ignition.math.Vector3.Distance() function and output the distance value.
```
Comment (chapulina) [inline tutorials/pythongetstarted.md:59]:
```suggestion
`ignition.math.Vector3.distance()` function and output the distance value.
```

**Review by chapulina — COMMENTED** [review]
Just minor comments

**Review by chapulina — APPROVED** [review]

**Comment by osrf-triage** [comment]
This pull request has been mentioned on **Gazebo Community**. There might be relevant details there:

https://community.gazebosim.org/t/new-ignition-releases-2022-03-01-citadel-edifice-fortress/1313/1

**Pull request merged**